### DISCUSSION TITLE
posts/2024-02-06-cytokines2/

### DISCUSSION THREAD

**Discussion by giscus** [racheltho/rachel-blog#8] [Announcements]
# posts/2024-02-06-cytokines2/

Immune cells are like a social network. Both math and benchtop experiments are useful in studying them.

https://rachel.fast.ai/posts/2024-02-06-cytokines2/

**Comment by yvesnewman** [comment]
Great 2nd part! Will there be a 3rd part coming, if requested?

**Reply by racheltho** [reply]
Thanks! I am currently working on a post about additional areas where AI can be applied to immunology, as well as a post on medical AI ethics. If you have questions about immune cell-cytokine networks, I am open to writing about the topic more!
